### ISSUE TITLE
GLITCH with chat option

### ISSUE THREAD

**Issue by fsalam90** [Crymzix/Pretzel#57]
The chat doesn't work immediately ..It doesn't update message that are being sent simultaneously. Please look into that!

**Comment by Crymzix** [comment]
Could you please clarify this with an example?

**Comment by fsalam90** [comment]
For example, in Whatsapp, when someone is messaging me and I have the chat window open, I will see it in real time. However in this case, even when the Window is open, and someone is messaging me, It appears as a notification, and I have to exit the chat and reopen it to see the message. It is not real time.

**Comment by Crymzix** [comment]
Yes, this is a limitation of the backend. Real-time messaging is a whole other ball game. I would have to integrate another API into the backend to get real time messaging, either something like Layer, Firebase or Sinch. This would definitely push the deadline back but I don't think this is a huge issue as of now as the majority of people would just message someone and then close the app. The messaging implementation being used right now is pretty basic but of course, as development progresses this will be a feature that the app should have.

**Comment by fsalam90** [comment]
Noted!
